PyRoute/Position/Hex.py

```diff
@@ -0,0 +1,97 @@
+"""
+Created on Aug 29, 2023
+
+@author: CyberiaResurrection
+"""
+import functools
+
+
+class Hex(object):
+
+    position = None  # Hex location
+    row = 0  # Location in the sector.
+    col = 0
+    dx = 0  # location in the whole space, row/column coordinate
+    dy = 0
```

Comment:
As I recall, but still need to dig into this, but we may not need the dx/dy value. In the original version of this, the x/y were used to calculate distances and the q/r values were used by the border drawing. But if you're using the q/r for distance calculations, the dx/dy are unused and you should be able to remove them. 

See also updating the Star class to move the q/r uses entirely here.

Reply:
I'll leave it in unless/until you decide it needs to go bouncing down the road.